src/daily_output.ts

```diff
@@ -290,13 +292,13 @@ export class DailyOutput {
 
 			const daysDiff = daysDiffInEST(previousDate, queryDate);
 			for (let index = 0; index < daysDiff; index++) {
-				console.log(`Prev block: ${previousBlockHeight}, current block: ${height}`);
+				console.log(`Prev block: ${this.data.blockHeight}, current block: ${height}`);
```

Comment:
What are the differences between `this.data.blockHeight`, `height`, and `previousBlockHeight` at this scope?

Reply:
I might have to rename those, but...
- `previousBlockHeight` is the height written on the daily_output.json at the previous run
- `height` is the currently iterated height
- `this.data.blockHeight` is the block iterated before the current one

Reply:
Can you rename them? It's very confusing

Reply:
[39c4c21](https://github.com/ardriveapp/inferno-backend/pull/27/commits/39c4c21886000f1ad01652ffc8b74497d9edb773)